config/owasp/suppressions.xml

```diff
@@ -1,95 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.3.xsd">
-  <suppress until="2022-08-01">
-    <notes>groovy-all</notes>
-    <cve>CVE-2020-17521</cve>
-  </suppress>
-  <suppress until="2022-08-01">
-    <notes>spring-boot-starter-oauth2-client</notes>
-    <cve>CVE-2018-1258</cve>
-    <cve>CVE-2021-22112</cve>
-  </suppress>
-  <suppress until="2022-08-01">
-    <notes>cpe</notes>
-    <cve>CVE-2018-1258</cve>
-    <cve>CVE-2021-22112</cve>
-  </suppress>
-  <suppress until="2022-08-01">
-    <notes>spring-statemachine-test</notes>
-    <cve>CVE-2013-6429</cve>
-    <cve>CVE-2013-6430</cve>
-    <cve>CVE-2014-0225</cve>
-    <cve>CVE-2014-1904</cve>
-    <cve>CVE-2016-9878</cve>
-    <cve>CVE-2018-1270</cve>
-    <cve>CVE-2018-1271</cve>
-    <cve>CVE-2018-1272</cve>
-    <cve>CVE-2020-5421</cve>
-  </suppress>
   <suppress until="2022-12-01">
-    <cve>CVE-2013-4152</cve>
-    <cve>CVE-2013-7315</cve>
-    <cve>CVE-2014-0054</cve>
-    <cve>CVE-2022-22965</cve>
-    <cve>CVE-2022-22968</cve>
+    <notes>This vulnerability still exists but NVD informs that this vulnerability has been modified and is currently undergoing reanalysis.</notes>
     <cve>CVE-2016-1000027</cve>
   </suppress>
```

Comment:
I know this vulnerability. It applies only to serialization/deserialization in the Java "native" way, which we don't use.

This vulnerability is not applicable to us. You can put a much larger timeframe (I suggest 2023-12-31) and state that we are not affected and that there is an expiry only to ensure we check from time to time that we are still unaffected.